py/torch_tensorrt/dynamo/backend/backends.py

```diff
@@ -77,13 +77,12 @@ def _pretraced_backend(
         )
         return trt_compiled
     except:
-        logger.error(
-            "FX2TRT conversion failed on the subgraph. See trace above. "
-            + "Returning GraphModule forward instead.",
-            exc_info=True,
-        )
-
         if not settings.pass_through_build_failures:
+            logger.error(
+                "FX2TRT conversion failed on the subgraph. See trace above. "
+                + "Returning GraphModule forward instead.",
+                exc_info=True,
+            )
```

Comment:
Only log an error if the user is not requesting `pass_through_build_failures`. If they are requesting `pass_through_build_failures`, the error is redundant since the compilation will halt anyway.

Reply:
Drop uses of the term FX2TRT